beacon-node-oapi.yaml

```diff
@@ -311,7 +315,7 @@ components:
     Bellatrix.SignedBlindedBeaconBlock:
       $ref: './types/bellatrix/block.yaml#/Bellatrix/SignedBlindedBeaconBlock'
     ConsensusVersion:
-      enum: [phase0, altair, bellatrix, capella]
+      enum: [phase0, altair, bellatrix, capella, eip4844]
```

Comment:
I think i'll avoid mentioning it more unless we miss something, but they should all just reference `deneb` where `eip4844` currently referenced

Reply:
hey @rolfyone I think you might be looking at earlier commits (before latest master was merged). I'm not seeing any 4844 references on this branch, might have to change the file filter to all commits in the diff view? 
Almost got me though 😅

Reply:
oh! weird i'll check

Reply:
looks right, im not sure how i got to an old commit :/